src/docs/product/integrations/feature-flag/launchdarkly/index.mdx

```diff
@@ -0,0 +1,22 @@
+---
+title: LaunchDarkly
+sidebar_order: 1
+
+description: Learn about Sentry's LaunchDarkly integration. LaunchDarkly enables organizations to use Sentry errors as a metric in their LaunchDarkly experiments.
+---
+
+LaunchDarkly enables organizations to use Sentry errors as a metric in their LaunchDarkly experiments.
+
+This integration is maintained and supported by the company that created it. For more details, check out our [Integration Platform documentation](/product/integrations/integration-platform/).
+
+## Install and Configure
+
+<Note>
+
+Sentry owner or manager permissions are required to install this integration.
+
+</Note>
+
+1. Navigate to **Settings > Integrations > LaunchDarkly**
+
+2. Follow the full [LaunchDarkly installation instructions](https://docs.launchdarkly.com/integrations/sentry).
```

Comment:
Are you sure this is the right link? I get a 404 error

Reply:
It's not live yet on their end it will be up shortly.